edxadapt/edxadapt.py

```diff
@@ -100,16 +120,24 @@ def student_view(self, context=None):
         html = self.resource_string("static/html/student_view.html")
         anonymous_student_id = self.get_anonymous_student_id()
         frag = Fragment(html.format(
-            self=self,
             anonymous_student_id=anonymous_student_id,
-            student_is_registered=self.student_is_registered,
-            course_id=self.get_course_id(),
-            edx_adapt_api_url=self.edx_adapt_api_url.rstrip('/'),
-            params=self.params,
-            skills=self.skills
+            success_registration_msg=self.success_registration_msg,
+            fail_registration_msg=self.fail_registration_msg,
         ))
         frag.add_css(self.resource_string("static/css/edxadapt.css"))
-        frag.add_javascript(self.resource_string('static/js/src/edxadapt.js'))
+        frag.add_javascript(
+            html_parser.unescape(
+                Template(self.resource_string(
+                    'static/js/src/edxadapt.js'
+                )).render(Context({
+                    'anonymous_student_id': anonymous_student_id,
+                    'course_id': self.get_course_id(),
+                    'edx_adapt_api_url': self.edx_adapt_api_url.rstrip('/'),
+                    'params': self.params,
```

Comment:
Hold on. I though we've moved `params` and `skills` into html template. Looks like merging went wrong.

Or do I miss something?